src/SpineCenterline.jl

```diff
@@ -9,13 +9,14 @@ Reconstruct the coil sensitivity map using the MRIReco.jl function and save it i
 # Arguments
 * `path_nifti::String` - path of the nifti file. The file must have .nii extension
 * `path_rep::String` - path of reference data in ISMRMRD format
+* `tresh::Float64` - masking threshold: increase for reduced mask size, decrease for extended mask size
 
 MRIReco reference: https://onlinelibrary.wiley.com/doi/epdf/10.1002/mrm.28792
 ISMRMRD reference: https://onlinelibrary.wiley.com/doi/epdf/10.1002/mrm.26089
 """
-function ReconstructSaveMap(path_nifti::String, path_ref::String)
+function ReconstructSaveMap(path_nifti::String, path_ref::String, thresh = 0.13)
 
-    (img, acq) = ReconstructMap(path_ref)
+    (img, acq) = ReconstructMap(path_ref, thresh)
```

Comment:
same as above

Reply:
@alexjaffray I think we didn't add tests on these two functions initially because they simply just call other functions that are covered by the tests.